ONSdigital/census-rm-ddl
# Add script to create read access role for RM tables

# Motivation and Context
We need to be able to create users with only read access on RM tables.

# What has changed
* Add script to create read access role for RM tables

# How to test?
Run the script then create a new user and then grant them `rm_read_access` role with 
```sql
GRANT rm_read_access TO <user>;
```
Then try connecting to the database as this user. You should be able to run `SELECT`'s on the RM schema tables and nothing else.

# Links
https://trello.com/c/WAn7nBms/1122-create-read-only-database-role-3
